src/cow-react/modules/wallet/api/utils/connection.ts

```diff
@@ -48,3 +50,11 @@ export function getIsZengoWallet(name: string | undefined): boolean {
 export function getIsAlphaWallet(name: string | undefined): boolean {
   return name?.toLocaleLowerCase().includes('alpha') || false
 }
+
+export function getIsWallet(name: string | undefined): boolean {
```

Comment:
Looks same with getIsAlphaWallet, isn't?

Reply:
Sorry, I don't understand, why do we need two exactly the same methods with different names?

Reply:
Oh my mistake, we don't need `getIsWallet` of course.